### PR TITLE
feat(page-filters): Add page filters doc section

### PR DESCRIPTION
**Pull request by davidenwang**
image

### CODE REVIEW

File: src/docs/product/sentry-basics/search/index.mdx [hunk 1]
```diff
@@ -10,6 +10,16 @@ description: "Learn more about Search, which is available in several Sentry feat
 
 Search is available on several features throughout [sentry.io](https://sentry.io), such as **Issues**, **Discover** and **Dashboards**.
 
+## Filters
+
+Page filters enable you to narrow down the results shown on a page by restricting data to a certain project, environment, or date range.
+
+  ![Page Filters](search-page-filters.png)
+
+These filters are normally only persistent within the current page, but you can choose to "lock" the current filter selection such that any changes made to the "locked" filter will be persisted throughout the application and even remembered between sessions.
```
Comment (imatwawana) [inline src/docs/product/sentry-basics/search/index.mdx:19]: This sentence is pretty long so I've broken it up. I also added a couple words to speak to current and change behaviour.

```suggestion
Typically, these filters only persist within the current page, but you can choose to "lock" the current filter selection. When you do this, the locked filter and any changes to it are persisted throughout the application and remembered between sessions. 
```

File: src/docs/product/sentry-basics/search/index.mdx [hunk 2]
```diff
@@ -10,6 +10,16 @@ description: "Learn more about Search, which is available in several Sentry feat
 
 Search is available on several features throughout [sentry.io](https://sentry.io), such as **Issues**, **Discover** and **Dashboards**.
 
+## Filters
+
+Page filters enable you to narrow down the results shown on a page by restricting data to a certain project, environment, or date range.
```
Comment (imatwawana) [inline src/docs/product/sentry-basics/search/index.mdx:15]: We avoid the word enable unless we're talking directly about enabling a setting 

```suggestion
Page filters allow you to narrow down the results shown on a page by restricting data to a certain project, environment, or date range.
```

File: src/docs/product/sentry-basics/search/index.mdx [hunk 3]
```diff
@@ -10,6 +10,16 @@ description: "Learn more about Search, which is available in several Sentry feat
 
 Search is available on several features throughout [sentry.io](https://sentry.io), such as **Issues**, **Discover** and **Dashboards**.
 
+## Filters
```
Comment (imatwawana) [inline src/docs/product/sentry-basics/search/index.mdx:13]: The challenging content for users to understand on this page is the syntax, so i don't want that get buried below the fold. I think we should move this section to the end (just before Learn More). 
```suggestion
## Page Filters
```

File: src/docs/product/sentry-basics/search/index.mdx [hunk 4]
```diff
@@ -10,6 +10,16 @@ description: "Learn more about Search, which is available in several Sentry feat
...
+
+Page filters enable you to narrow down the results shown on a page by restricting data to a certain project, environment, or date range.
+
+  ![Page Filters](search-page-filters.png)
+
+These filters are normally only persistent within the current page, but you can choose to "lock" the current filter selection such that any changes made to the "locked" filter will be persisted throughout the application and even remembered between sessions.
+
+  ![Page Filters Locking](search-page-filters-lock.png)
+
 ## Syntax
```
Comment (imatwawana) [inline src/docs/product/sentry-basics/search/index.mdx:23]: I think it'll be helpful to make this more descriptive in light of the fact that we're adding content about page-level filters. 
```suggestion
## Query Syntax {#syntax}
```

**Review by imatwawana — COMMENTED** [review]
I've made some editing suggestions for the wording/styling as well as a structural suggestion. Let me know what you think.

**Comment by davidenwang** [comment]
Thanks for the suggestions!

**Review by evanpurkhiser — APPROVED** [review]
This feels good to me

**Review by imatwawana — COMMENTED** [review]
The content looks good! I just think the syntax is the challenging content on this page, so it should come first. I think we should move the Page Filters section to after the Query Syntax content (so just before the Learn more section).

**Pull request merged**